### ISSUE TITLE
ssl_certificate.lua failed to validate OCSP

### ISSUE THREAD

**Issue by Cyrille37** [Valian/docker-nginx-auto-ssl#36]
Hello

Certificates failed renew after expiration because of OCSP failed
```
[lua] ssl_certificate.lua:260: set_response_cert():
auto-ssl: failed to set ocsp stapling for www.domain.com - continuing anyway -
failed to get ocsp response: failed to validate OCSP response (http://ocsp.int-x3.letsencrypt.org):
OCSP response not successful (6: unauthorized),
context: ssl_certificate_by_lua*, client: xxx.xxx.xxx, server: 0.0.0.0:443
```

I found on Internet that Certbot 0.39.0 will also prevent this error from appearing in the future ; it was updated to not send OCSP requests for certificates that are expired.

**Comment by Cyrille37** [comment]
Please, can you help me about this blocking issue ?
I'm using "valian/docker-nginx-auto-ssl latest" and cannot found how to force renew of certificates...

**Comment by Cyrille37** [comment]
The workaround is to delete the resty-auto-ssl data folder, that make it to recreate certificates. Docker-compose start and certificates renewed.

**Comment by Valian** [comment]
Is there anything I can change in the repo to help with it? I'm pretty sure the latest version of this image should work correctly, without the need to recreate volume.

**Comment by Cyrille37** [comment]
hello. I don't know how this happened.
Perhaps no http request for few days where the certificate expired. Then a request that make openresty asking for a renew but failed because OSCP failed because expired certificate.
I found this subject while search on internet, with posts say it's resolved with Certbot 0.39.0, perhaps the LetsEncrypt library used by openresty should be updated.
But I really don't know, it is to far from my knowledge.
Happily, I'm not in production, just in qualification, so I can live with this problem...

**Comment by Valian** [comment]
This image has been updated to the latest version of lua-resty-auto-ssl, hopefully this shouldn't happen again. If someone has a similar problem, just try to pull the image again, stop the container, remove certificates from volume and create a container with the newest version. It should work again.
